src/data/faq.json

```diff
@@ -2809,7 +2809,7 @@
                                 "anchor": "update_faq",
                                 "active": false,
                                 "textblock": [
-                                    "The last content changes to FAQ articles were made on February 21, 2023.",
+                                    "The last content changes to FAQ articles were made on March 20, 2023.",

```

Comment:
@larswmh 

It seems that this item is often forgotten when FAQs are updated.

It is probably too late now to automate this, but one idea would be to take the last commit date from git and add it somewhere else, like into the footer.

```
$ git log -1 --pretty="format:%cs" src/data/faq_de.json
2023-03-15
```

Reply:
This is indeed often forgotten about and can arguably be seen as a bad solution for this problem in general, but I would still consider it as better than the solution before that just put the date of deployment there (regardless of whether there were changes of the FAQ or not). 

Although I'm not satisfied with the implementation either, I'd suggest to leave it as it is for the remaining timespan. As we usually do not delete FAQ articles for archiving purposes, it was agreed that this article about the latest FAQ update date should remain.